go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/ratify-project/ratify-go
+
+go 1.22
```

Comment:
since ratify is already on 1.23, I think we can start from 1.23 in this repo

Reply:
It is better to work on 1.22 for libraries that's N-1 of go version.